docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+|                     |                      | Affiliation name            | affiliation_name             | STRING                                  |  --  |   --    |                    RECOMMENDED                    | ROR                                   | Name of the institution an author is affiliated with. Sometimes, one author may have multiple affiliations                                                                                               |
+|                     |                      | Affiliation address         | affiliation_address          | STRING                                  |  --  |   --    |                     OPTIONAL                      | N/A                                   | Address of the institution an author is affiliated with                                                                                                                                                  |
+|                     |                      | Affiliation identifier      | affiliation_identifier       | STRING                                  |  --  |   --    |                    RECOMMENDED                    | ROR                                   | A unique identifier assigned to the affiliated institution by The Research Organization Registry (ROR)                                                                                                   |
+|                     |                      | Author list order           | order                        | INTEGER                                 |  --  |   --    |                       MUST                        | N/A                                   | The order in which the author appears in the publication                                                                                                                                                 |
+| Funding             | funding              | Funding agency name         | funding_agency_name          | STRING                                  |  --  |   --    |                    RECOMMENDED                    | ROR or Crossref Funder Registry       | Name of the funding agency. There could be multiple funding agencies                                                                                                                                     |
+|                     |                      | Grant ID                    | grant_id                     | STRING                                  |  --  |   --    |                    RECOMMENDED                    | N/A                                   | Grant identifier provided by the funding agency                                                                                                                                                          |
```

Comment:
The entire Funding section is actually not a dictionary, but a list of Dictionaries of type "DatasetFunding". We should reflect this in the documentation.